saleor/page/migrations/0030_slug_translation_unique_constraint.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.postgres.indexes import BTreeIndex
+from django.contrib.postgres.operations import AddIndexConcurrently
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    atomic = False
+
+    dependencies = [
+        ("page", "0029_add_page_translation_slug"),
+    ]
+
+    operations = [
+        AddIndexConcurrently(
+            model_name="pagetranslation",
+            index=BTreeIndex(
+                fields=["language_code", "slug"], name="pagetranslation_slug_idx"
+            ),
+        ),
+        migrations.AddConstraint(
```

Comment:
And this is not covered in example. but this part you will need to add to the `state` in `migrations.SeparateDatabaseAndState`.

Reply:
Thanks for the guidance! Should look better now